runtime/queries/hocon/highlights.scm

```diff
@@ -0,0 +1,33 @@
+(comment) @comment
+
+(null) @constant.builtin
+[(true) (false)] @constant.builtin.boolean
+(number) @constant.numeric
+(string) @string
+(multiline_string) @string
+(string (escape_sequence) @constant.character.escape)
+(unquoted_string) @string
+
+(value [":" "=" "+=" ] @operator)
+
+(substitution (_) @string)
+(substitution ["${" "${?" "}"] @punctuation.special)
+
+[ 
+  "url"
+  "file"
+  "classpath"
+  "required"
+] @function.builtin
+
+(include "include" @include)
+
+[ "(" ")" "[" "]" "{" "}" ]  @punctuation.bracket
+
+(unit) @keyword
+(path (_) @keyword)
+(unquoted_path "." @punctuation.delimiter)
+[ "," ] @punctuation.delimiter
+
+(ERROR) @error
+
```

Comment:
```suggestion
```

eventually we'd like to make syntax error highlighting a configuration option. For now we remove it from highlight queries since it's usually very distracting while typing